test/generate-peers/generate-peer.go

```diff
@@ -0,0 +1,85 @@
+package main
```

Comment:
I wonder if we really want this in the tests, or a top level folder

Reply:
I think it's useful to show we're working with the same data